src/components/nav/OtherRoutes.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+import { HomePage } from "../homePage/HomePage";
+import { SearchBar as VolunteeringSearchBar } from "../volunteering/SearchBar";
+import { FacebookLoginLogout } from "../facebook/FacebookLoginLogout";
+import { CastleRisk } from "../castleRisk/CastleRisk";
+import { Router } from "./Router";
+
+export const OtherRoutes = (url, props, router: Router) => router
```

Comment:
these "other routes" would eventually be moved closer to their containing folders, for now I just wanted to separate out "org routes" and so needed a similar way to define all the other, non-org routes, and this is it.